cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -100,5 +178,4 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
 
         Files.write(dockerComposePath, dockerComposeContent.toByteArray(StandardCharsets.UTF_8))
```

Comment:
Kotlin's default `Charset` is already `UTF-8` and so you don't need to set this here.